.travis.yml

```diff
@@ -1,29 +1,27 @@
 language: c
 
 sudo: false
-cache:
-  directories:
-  - $HOME/.stack/
 
 matrix:
   include:
     - env: CABALVER=1.22 GHCVER=7.8.4
       addons: {apt: {packages: [cabal-install-1.22,ghc-7.8.4], sources: [hvr-ghc]}}
-    - env: CABALVER=1.22 GHCVER=7.10.3
-      addons: {apt: {packages: [cabal-install-1.22,ghc-7.10.3],sources: [hvr-ghc]}}
+    - env: CABALVER=1.22 GHCVER=7.10.2
+      addons: {apt: {packages: [cabal-install-1.22,ghc-7.10.2],sources: [hvr-ghc]}}
```

Comment:
Ah no, I think I was faffing to get it working and made a mistake. I'll change to .3